storage/mysql/log_storage.go

```diff
@@ -92,7 +94,16 @@ func NewLogStorage(id int64, dbURL string) (storage.LogStorage, error) {
 	// TODO: This should not default but it would currently complicate testing and can be
 	// implemented later when the create tree API has been defined.
 	if err := s.db.QueryRow(getTreePropertiesSQL, id).Scan(&s.allowDuplicates); err == sql.ErrNoRows {
-		s.allowDuplicates = false
+		// Create new Tree.
```

Comment:
I'm not convinced that magically auto-creating a `Trees` entry is a good idea; I think @codingllama is planning things around log provisioning.